spec/origin_info_spec.rb

```diff
@@ -0,0 +1,146 @@
+describe "computations from /originInfo field" do
+
+  let(:smods_rec) { Stanford::Mods::Record.new }
+
+  let(:mods_origin_info_start_str) { "<mods xmlns=\"#{Mods::MODS_NS}\"><originInfo>" }
```

Comment:
Ok. I'm just afraid we're introducing two different ways of doing the same thing.